CMakeLists.txt

```diff
@@ -115,7 +115,11 @@ endif()
 if(NOT DISABLE_CERN_ROOT)
   find_package(CERN_ROOT)
   if (CERN_ROOT_FOUND)
-    message(STATUS "ROOT Version: ${CERN_ROOT_VERSION}")
+    message(STATUS "ROOT Version: ${CERN_ROOT_VERSION_MAJOR}.${CERN_ROOT_VERSION_MINOR}.${CERN_ROOT_VERSION_PATCH}")
```

Comment:
This should work
```cmake
if (${CERN_ROOT_VERSION} VERSION_GREATER 6.23)
...
```
So no need for the parsing in the `Find*cmake`.
(`VERSION_GREATER_EQUAL` was introduced in 3.8 or so, while we only require 3.1).